src/internal/context/app-layout-context.ts

```diff
@@ -3,17 +3,10 @@
 import { useContext, createContext } from 'react';
 
 export interface AppLayoutContextProps {
-  stickyOffsetBottom: number;
-  stickyOffsetTop: number;
-  mobileBarHeight?: number;
   setHasStickyBackground?: (hasBackground: boolean) => void;
```

Comment:
Cannot remove this context entirely, because of this dependency. 

It will be gone in a separate PR with the entire dark header feature